docs/concepts/services-networking/service.md

```diff
@@ -561,6 +561,23 @@ In the above example, if the service contained three ports, `80`, `443`, and
 `8443`, then `443` and `8443` would use the SSL certificate, but `80` would just
 be proxied HTTP.
 
+Beginning in 1.9, services can use [predefined AWS SSL policies](http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-security-policy-table.html)
+for any HTTPS or SSL listeners. To see which policies are available for use, run
+the awscli command:
+
+```bash
+aws elb describe-load-balancer-policies --query 'PolicyDescriptions[].PolicyName'
+```
+
+Any one of those policies can then be specified using the following annotation:
```

Comment:
(trying to be picky ...)

> using the following annotation:

using the "`service.beta.kubernetes.io/aws-load-balancer-ssl-negotiation-policy`" annotation, for example:

Reply:
No problem! Thanks